regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
Curious, do these need to be converted to booleans? Quick Googling doesn't give a definitive answer: http://stackoverflow.com/a/19234987 http://stackoverflow.com/a/24701778